### ISSUE TITLE
errors when checking for data types for ADD COLUMN events

### ISSUE THREAD

**Issue by FxKu** [pgMemento/pgMemento#12]
**TL;DR**: the `to_regtype` method in the event trigger procedure `table_alter_pre_trigger` throws either syntax errors or doesn't catch data type with spaces (= no log)

**Details**
The event trigger procedure `table_alter_pre_trigger` tries to parse different kinds of DDL statements that modifiy columns. The general idea is to split the DDL text by spaces into an array. Whenever the keywords `ADD`, `ALTER` or `DROP` are found within a loop over the array it is expected to find the column name in the next iteration or the one after the next. For `ALTER` and `DROP` events the extracted columnname can be check againt the `audit_column_log` table.

For ADD COLUMN events this is not possible as the column does not exist yet. Therefore, the current assumption is to look out for a data type in the next iteration. I thought, this could easily be checked with Postgres' `to_regtype` function. If the current text input resembles a data type `to_regtype` returns the name, else NULL. Unfortunately, splitting up the DDL text can produce input that cause syntax errors, as reported by @coutoPL in his PR #11.

What's even worse it that data types with spaces do not produce the error but can bypass the logging. For example a data type like `double precision` results in two iterations with `double` and `precision`. `to_regtype('double')` returns zero, so the ADD COLUMN event is not logged. There would also be no entry in the `audit_column_log` table.

**Comment by FxKu** [comment]
Should be solved with latest commit cc0f1ff

Anybody feel free to reopen, if you find other edge cases.
